### PR TITLE
Add a safearea around all the screens

### LINKED ISSUE #169: Make bottom padding consistent over all screens.
Some screens have various amounts of bottom padding when they are scrollable. It would be nice to make this consistent, perhaps using a SafeArea, so that there is always some space at the bottom of each screen.

### PR DESCRIPTION
**Pull request by JAicewizard**
### Summary
This adds a SafeArea around all the screens, ensuring enough space to avoid problems with the notches and other device/OS specific areas.

### CODE REVIEW

**Comment by DeD1rk** [comment]
Merging into a different branch so I can rebase this

File: lib/ui/screens/login_screen.dart [hunk 1]
```diff
@@ -29,59 +29,63 @@ class _LoginScreenState extends State<LoginScreen> {
...
-                  child: ElevatedButton(
-                    style: ElevatedButton.styleFrom(
-                      backgroundColor: Colors.black87,
-                      foregroundColor: Colors.white,
+                  const SizedBox(height: 50),
+                  SizedBox(
+                    height: 50,
+                    child: ElevatedButton(
+                      style: ElevatedButton.styleFrom(
+                        primary: Colors.black87,
```
Comment (DeD1rk) [inline lib/ui/screens/login_screen.dart:75]:
```suggestion
                        backgroundColor: Colors.black87,
```

File: lib/ui/screens/login_screen.dart [hunk 2]
```diff
@@ -29,59 +29,63 @@ class _LoginScreenState extends State<LoginScreen> {
...
+                        onPrimary: Colors.white,
```
Comment (DeD1rk) [inline lib/ui/screens/login_screen.dart:76]:
```suggestion
                        foregroundColor: Colors.white,
```

**Comment by DeD1rk** [comment]
CI can't deploy from a fork, so I'll merge into a different branch first.

**Comment by DeD1rk** [comment]
Or actually, I have no idea what I did but #281 should be able to build...

**Comment by JAicewizard** [comment]
Ok that's kinda annoying that it cant do that. Should I just use branches on this repo?

**Comment by DeD1rk** [comment]
> Ok that's kinda annoying that it cant do that. Should I just use branches on this repo?

Yup, I'll doouble-check that you have write access.

**Pull request closed without merging**